WrightTools/data/_channel.py

```diff
@@ -113,59 +104,17 @@ def signed(self):
             self.attrs['signed'] = False
         return self.attrs['signed']
 
-    def clip(self, min=None, max=None, replace='nan'):
-        """Clip (limit) the values in a channel.
-
-        Parameters
-        ----------
-        min : number (optional)
-            New channel minimum. Default is None.
-        max : number (optional)
-            New channel maximum. Default is None.
-        replace : {'val', 'nan'} (optional)
-           Replace behavior. Default is nan.
-        """
-        # decide what min and max will actually be
-        if max is None:
-            max = self.max()
-        if min is None:
-            min = self.min()
-        # replace values
-        if replace == 'val':
-            self[:].clip(min, max, out=self[:])
-        elif replace == 'nan':
-            self[self[:] < min] = np.nan
-            self[self[:] > max] = np.nan
-        else:
-            print('replace not recognized in channel.clip')
-
-    def invert(self):
-        """Invert channel values."""
-        self[:] *= -1
-
     def mag(self):
         """Channel magnitude (maximum deviation from null)."""
         return self.major_extent
 
-    def normalize(self, axis=None):
+    def normalize(self):
```

Comment:
What if we wish to normalize along a particular axis?

Reply:
yeah, I was disapointed to do this, but I think it just needs to be thought about

I'll open an issue

Reply:
Perhaps add it to #309